modules/siddhi-extensions/kf/src/main/resources/geo.siddhiext

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
Filename cannot be 'geo.siddhiext' since geo extension already exists and we are putting Kalman filter to a separate domain.